get-qualify-export-donor-data/anonymize-and-export-data/anonymize-and-export.py

```diff
@@ -675,5 +675,5 @@ def exportData(df, fileName, fileType, exportDirectory, mergeCalculatorData):
 else:
     outputName = "PHI-" + userID
 
-exportData(data, outputName, args.exportFormat,
+exportData(data, outputName, ast.literal_eval(args.exportFormat),
```

Comment:
@pazaan this addition breaks the code, I get a ValueError: malformed node or string: ['all']. I think that we either require the input args.exportFormat to be a list (as it is currently set up) OR, we add in the conditionals so that it can sense whether the input is a string or a list. I have a preference for the former, but am open to either, and am open to the idea that I missed something.